demos/common/python/monitors_extension/monitors_extension.cpp

```diff
@@ -96,7 +96,12 @@ PyObject *presenter_reportMeans(PresenterObject *self, PyObject *Py_UNUSED(ignor
         return nullptr;
     }
     try {
-        return PyUnicode_FromFormat(self->_presenter->reportMeans().c_str());
+        PyObject *PList = PyList_New(0);
+        auto report = self->_presenter->reportMeans();
+        for(const auto& s : report){
+            PyList_Append(PList, Py_BuildValue("s", s.c_str()));
+        }
+        return PList;
```

Comment:
I believe this should be in @anzhella-pankratova's patch because that's where people pay attention to how python prints are formatted.

Reply:
But you need `reportMeans()` to return a vector... Never mind